### PR TITLE
refactor(swap): move @yoroi/openswap to @yoroi/swap

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-1161](https://emurgo.atlassian.net/browse/YOMO-1161)

[YOMO-1161]: https://emurgo.atlassian.net/browse/YOMO-1161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/swap/src/adapters/openswap-api/api.ts
Comment (banklesss) [inline packages/swap/src/adapters/openswap-api/api.ts:1]: changed `api.ts` to `api-maker.ts`

and added `api.ts` from `@yoroi/openswap` here

**Comment by jorbuedo** [comment]
Does the removal of the openswap package affect mobile or extension using it? 
I guess it was only used inside the swap package, but just wondering if we need some major version change or not.

**Comment by banklesss** [comment]
> Does the removal of the openswap package affect mobile or extension using it?
I guess it was only used inside the swap package, but just wondering if we need some major version change or not.

I've been checking and it was only being used internally in the swap package, I will double-check JIC

**Review by jorbuedo — APPROVED** [review]
Seems to be working fine

**Pull request merged**